tests/test_efdUtils.py

```diff
@@ -237,6 +237,18 @@ def test_clipDataToEvent(self):
 
         self.assertGreaterEqual(dataStart, self.event.begin)
         self.assertLessEqual(dataEnd, self.event.end)
+
+        # test the pre/post padding options
+        clippedPaddedData = clipDataToEvent(dayObsData, self.event, prePadding=1, postPadding=2)
+        self.assertIsInstance(clippedPaddedData, pd.DataFrame)
+        self.assertGreater(len(clippedPaddedData), 0)
+        self.assertLess(len(clippedPaddedData), len(dayObsData))
+        self.assertGreater(len(clippedPaddedData), len(clippedData))
+
```

Comment:
Again, I had the same thought. It's actually quite easy to check in this instance, with hard-coded test data, but it's hard in the general case, because a) it's not going be exact, due to the quantized nature of the data, and b) it'll depend on whether or not the input data actually extends out far enough (hence the warning logs for when it doesn't).

Reply:
I'll take a swing at that in the morning though, thanks.

Reply:
OK, pushed a fix for this, it's in Jenkins now.